splinterd/man/splinterd.1.md

```diff
@@ -112,6 +112,13 @@ OPTIONS
   Any options on the command line will override the settings in the
   configuration file.
 
+`--config-dir CONFIG_DIR`
+: Specifies the directory containing configuration files. This is appended to
+  the directory containing certificates and associated key files if an absolute
```

Comment:
Technically "absolute path" is one that starts with "/", so this should be stated in a different way.